docs/variables.md

```diff
@@ -1595,9 +1676,9 @@ Enabled by default.
 | Data type    | Boolean                |
 | Default      | TRUE                   |
 
-When enabled, this option allows index key prefixes longer than 767 bytes (up to
-3072 bytes). The values for rocksdb_large_prefix should be the same between
-source and replica.
+This variable is deprecated in `Percona Server for MySQL 8.0.36-28` and will be removed in a future release.
```

Comment:
Do we have an alternative variable - what they should use instead of?

Reply:
We do not have an alternative variable. Approved by Yura.